java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -511,6 +513,49 @@ public void getdel() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void getex() {
+        // setup
+        String key = "testKey";
+        String value = "testValue";
+        CompletableFuture<String> testResponse = new CompletableFuture<>();
+        testResponse.complete(value);
+        when(commandManager.<String>submitNewCommand(eq(GetEx), eq(new String[] {key}), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<String> response = service.getex(key);
+        String payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(value, payload);
+    }
+
+    @SneakyThrows
+    @Test
+    public void getexOptions() {
```

Comment:
nit: maybe follow the snake case naming that the rest of the tests are using (something like `getex_options`)